tests/translator/input/api_with_resource_policy_global_intrinsic.yaml

```diff
@@ -0,0 +1,26 @@
+Globals:
```

Comment:
This file should be renamed: `api_with_resource_policy_global_intrinsic` -> `api_with_resource_policy_global_implicit`